mail-client/bower/bower-0.9.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A curses terminal client for the Notmuch email system"
+HOMEPAGE="https://github.com/wangp/bower"
+SRC_URI="https://github.com/wangp/bower/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="app-crypt/gpgme
```

Comment:
Why is gpgme used only at build time? Is this some ugly static linking kind of thing?

Reply:
I'm taking a queue from the documented requirements here as well. I'm not certain what the author does once the software is running, and I personally think that decryption should be delegated to Notmuch anyway.

Reply:
Figure it out. You have all the readelfs, etc. You can see what's it linking, how. While at it, make sure that mercury is not runtime dep due to some runtime as well.

Reply:
Fixed RDEPEND.

Reply:
gpgme is subslotted, so you need `:=` dep.

Reply:
Good point, fixed.